content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -0,0 +1,202 @@
+# Sharing Experiments
+
+There are two types of remotes that can store experiments. Git remotes are
+distributed copies of the Git repository, for example on GitHub or GitLab.
+
+[DVC remotes](/doc/command-reference/remote) on the other hand are
+storage-specific locations (e.g. Amazon S3 or Google Drive) which we can
+configure with `dvc remote`. DVC uses them to store and fetch large files that
+don't normally fit inside Git repos.
+
+DVC needs both kinds of remotes for backing up and sharing experiments.
+
+Experiment files that are normally tracked in Git (like code versions) are
+shared using Git remotes, and files or directories tracked with DVC (like
+datasets) are shared using DVC remotes.
+
+> See [Git remotes guide] and `dvc remote add` for information on setting them
+> up.
+
+[git remotes guide]:
+  https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
+
+Normally, there should already be a Git remote called `origin` when you clone a
+repo. Use `git remote -v` to list your Git remotes:
+
+```dvc
+$ git remote -v
+origin  https://github.com/iterative/get-started-experiments (fetch)
+origin  https://github.com/iterative/get-started-experiments (push)
+```
+
+Similarly, you can see the DVC remotes in you project using `dvc remote list`:
+
+```dvc
+$ dvc remote list
+storage https://remote.dvc.org/get-started-experiments
+```
+
+_Q: What about neither, `cache: false` objects, and objects tracked both by DVC
+and Git?_
```

Comment:
Was this a WIP kind of note?

Reply:
Yes, it is. I'm trying the commands here and wondered how they behave in these situations. Let's remove this for now.